frontend/src/components/TopBar.tsx

```diff
@@ -0,0 +1,77 @@
+import { MFALogoIcon, ProfileIcon, SearchIcon } from "@/static/icons";
+import {
+  Box,
+  Hide,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Select,
+  Text,
+  Flex
+} from "@chakra-ui/react";
+
+export const TopBar: React.FC = () => {
+  return (
+    <Flex
+      as="header"
+      align="center"
+      pos="fixed"
+      w="full"
+      px="4"
+      bg="white"
+      _dark={{
+        bg: "gray.800",
+      }}
+      borderBottomWidth="1px"
+      color="inherit"
+      h="136"
+    >
+      <Box minWidth={302}>
```

Comment:
This looks awesome! My only main suggestion is to maybe add a breakpoint to hide the MFA logo as the width of the window decreases, similar to the "Welcome back, user". Currently, the search bar gets really weird and squished when the window width decreases and I feel like the logo isn't as important as having the search bar look correct